src/components/Cart/Cart.tsx

```diff
@@ -0,0 +1,47 @@
+import './Cart.scss';
+import LeftIcon from '../../icons/left.svg';
+import { NavLink, useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+
+export const Cart = () => {
+  const [cartItems, setCartItems] = useState([]);
+  const navigate = useNavigate();
+
+  const handleCheckout = () => {
+    const userConfirmed = confirm(
+      'Checkout is not implemented yet. Do you want to clear the Cart?',
+    );
+
+    if (userConfirmed) {
+      setCartItems([]);
+      navigate('/phones');
+    }
+  };
+
+  return (
+    <>
+      <div className="cart">
+        <NavLink to="#">
+          <img className="cart__icon" src={LeftIcon} alt="left" />
```

Comment:
In mockup button is black instead of grey

Reply:
color changed